manifests/02_config.cr.yaml

```diff
@@ -4,5 +4,7 @@ metadata:
   name: cluster
   annotations:
     release.openshift.io/create-only: "true"
+    # this flag is not set for a cluster coming from 4.5 via upgrade. Hence, 4.5 clusters will keep supporting non-sha256 tokens.
+    oauth-apiserver.openshift.io/secure-token-storage: "true"
```

Comment:
Is 'fresh 4.6 install' a must requirement? Can we configure 'oauth-apiserver.openshift.io/secure-token-storage: "true"' manually after upgrading OCP4.5 to OCP4.6?

Reply:
If you can guarantee that you don't have old oauth tokens (those without `sha256-` prefix) in the system, yes you can technically set this annotation.

Reply:
Thanks for your comment. I will try to test this patch and feed back later.

Reply:
how is it going?

Reply:
Thanks. The patch works well. I confirmed we can monitor events of creating/deleting oauthaccesstokens. So we think we can monitor login/logout events using this feature. But we are testing this patch now and it will take some time to complete. Please wait a moment for further feedback.

Reply:
We don't have additional comments. This patch works fine as we expected. Thanks.